tools/memap.py

```diff
@@ -10,10 +10,11 @@
 import argparse
 from prettytable import PrettyTable
 
-from tools.utils import argparse_filestring_type, \
+from utils import argparse_filestring_type, \
     argparse_lowercase_hyphen_type, argparse_uppercase_type
 
 DEBUG = False
+DETAILED = False
```

Comment:
Global variables should not change, so that if we ever need to run many of these concurrently in the same process, we can. Could we make this an object variable?